doc/source/cluster/cloud.rst

```diff
@@ -272,6 +272,116 @@ There are two ways of running private clusters:
             $ ray down ray/python/ray/autoscaler/local/example-full.yaml
 
 
+.. _manual-cluster:
+
+Manual Ray Cluster Setup
+------------------------
+
+The most preferable way to run a Ray cluster is via the Ray Cluster Launcher. However, it is also possible to start a Ray cluster by hand.
+
+This section assumes that you have a list of machines and that the nodes in the cluster can communicate with each other. It also assumes that Ray is installed
+on each machine. To install Ray, follow the `installation instructions`_.
+
+.. _`installation instructions`: http://docs.ray.io/en/master/installation.html
+
+Starting Ray on each machine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On the head node (just choose some node to be the head node), run the following.
+If the ``--port`` argument is omitted, Ray will choose port 6379, falling back to a
+random port.
+
+.. code-block:: bash
+
+  $ ray start --head --port=6379
+  ...
+  Next steps
+    To connect to this Ray runtime from another node, run
+      ray start --address='<ip address>:6379' --redis-password='<password>'
```

Comment:
We need to tell people how to find the Redis password

Reply:
Can we get rid of mentions to `--redis-password`? we've been trying to deprecate it for a while now

Reply:
this is automatically output in the stdout of the command; we should fix the mentions in a future PR